pkg/clustertree/cluster-manager/runtime/controller.go

```diff
@@ -0,0 +1,123 @@
+package runtime
```

Comment:
It seems can move to https://github.com/kosmos-io/kosmos/blob/main/pkg/utils/informers/InformerFactory.go and https://github.com/kosmos-io/kosmos/blob/main/pkg/utils/controllers/controller_util.go
Can we merge into one package ？ 
@OrangeBao @duanmengkk